mcp-server-gitparser/src/mcp_server_gitparser/hybrid_routers/parsing.py

```
@@ -0,0 +1,124 @@
+
+from fastapi import APIRouter, HTTPException
+
+from mcp_server_gitparser.config import get_app_settings
+from mcp_server_gitparser.gitparser.gitbook import clean_gitbook_url, convert_gitbook_to_markdown
+from mcp_server_gitparser.gitparser.github import clean_github_url, convert_repo_to_markdown
+from mcp_server_gitparser.schemas import ConvertGitbookRequest, ConvertGithubRequest, ConvertResponse
+
+router = APIRouter(tags=["Parsing"])
+
+
+def generate_filename(url: str, extension: str = "md") -> str:
```

Ruff (0.14.14)

[warning] Missing docstring in public function

(D103)